detekt

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+BASEDIR="$(dirname "$0")"
+if [[ ! -f "$BASEDIR/detekt.jar" ]]; then
+  curl -sL -o "$BASEDIR/detekt.jar" $(curl -s https://api.github.com/repos/detekt/detekt/releases/latest | jq -r '.assets[].browser_download_url')
```

Comment:
Probably a good idea to version pin the detekt binary, given that we had an issue with `latest` and DataDog APM recently

Reply:
Yeah, Barry said the same. I'm going to change it to use the now latest, with an option to provide your own version